packages/client-adapters/src/adapters/lexicons/adapter.js

```diff
@@ -369,28 +369,34 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
     // if we still don't have a match, we can do a last ditch check without
     // any diacritics at all in those languages that support it
     if (!found) {
-      let lastAlt = []
+      const lastAlt = []

```

Comment:
this change should be reversed by merging in master, Irina fixed it for me, I needed to add the eslint hint to force this to let